app/javascript/actionsPanel/actionsPanel.js

```diff
@@ -0,0 +1,321 @@
+import { request } from '../utilities/http/request';
```

Comment:
You can use the webpack alias here
```suggestion
import { request } from '@utilities/http';
```